tstest/integration/vms/README.md

```diff
@@ -0,0 +1,91 @@
+# End-to-End VM-based Integration Testing
+
+This test spins up a bunch of common linux distributions and then tries to get
+them to connect to a
+[`testcontrol`](https://pkg.go.dev/tailscale.com/tstest/integration/testcontrol)
+server.
+
+
+## Running
+
+Due to facts and circumstances, this test only runs on Linux. This life decision
```

Comment:
Simplify this to:

"This test currently only runs on Linux."

Stick to short and factual basically always. Cutesy joking/editorializing can be for blog posts.